apps/crn-frontend/src/dashboard/__tests__/Dashboard.test.tsx

```diff
@@ -22,6 +22,7 @@ jest.mock('../../shared-research/api');
 jest.mock('../../network/teams/api');
 jest.mock('../../network/users/api');
 
+const time = new Date();
```

Comment:
This should be in state. This will mean that the time is set for as long as the app is open. @lctrt  is doing something similar

Reply:
I'll do that.

Reply:
changed that in the main component, left it like this in the test as that was the approach in other components.